backend/core/tests/management/test_watch_account_creation.py

```diff
@@ -0,0 +1,30 @@
+from io import StringIO
+from unittest import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from core.models.user import EmailDomain
+from core.tests.factories.user import UserFactory
+from core.utils.time import time_ago
+
+
+class WatchAccountCreationTestCase(TestCase):
+
+    @mock.patch("core.lib.discord.api.write_in_channel")
+    def test_cmd_watch_account_creation(self, write_in_channel_mock):
```

Comment:
You implemented only one positive (positive == it triggers the notification) test (nice one btw), but let's have both positive and negative cases.

E.g. 5 new domains 65 minutes ago, 4 new domains 5 minutes ago, running the command with `since_n_hours=1, n_account=5`, assert that no notification is sent